catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1004,34 +1002,34 @@ public CreateResponse create(CreateRequest createRequest)
                 }
                 metacard.setAttribute(new AttributeImpl(Metacard.SECURITY, itemPolicyMap));
             }
-            createReq.getProperties()
+            createRequest.getProperties()
                     .put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
 
             for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
-                createReq = plugin.processPreCreate(createReq);
+                createRequest = plugin.processPreCreate(createRequest);
```

Comment:
Re-assigning an input parameter, especially so far down in a method, is usually considered a bad practice as people tend to assume that input parameters are constant, which is probably what the original code was after when it was doing `createReq = createRequest`. Part of the problem here is that the method is too long and `createRequest` gets manipulated so many different places in the same method.

If we don't want to rework the code too much for now, could we just keep the `createReq` (and maybe giving it a better name, e.g., `processedRequest`?) and use that reference when we create the response at L1035?